### PR TITLE
some ui improvements and fixes

### PR DESCRIPTION
**Pull request by aligator**
- added config file
- use load owner and restaurant by id
- some refactorings

### CODE REVIEW

File: controllers/rest/user.go [hunk 1]
```diff
@@ -151,3 +153,27 @@ func (c *Controller) SetPaypalMailAddr(session session.Manager) http.HandlerFunc
...
+		id, err := strconv.Atoi(chi.URLParam(r, "id"))
+
+		if err != nil {
+			respond(w, r, http.StatusUnprocessableEntity, ErrInvalidNumberFormat.Error())
+			return
+		}
+
+		publicUser, err := c.userService.Get(uint64(id))
+
+		if err != nil && err == offer.ErrOfferNotFound {
```
Comment (dominikbraun) [inline controllers/rest/user.go]: Wrong error, should be `user.ErrUserNotFound` :D

File: services/user/types.go [hunk 2]
```diff
@@ -30,3 +30,9 @@ type Login struct {
 type PaypalMailAddrSetter struct {
 	PaypalMailAddr string `json:"paypal_mail_addr"`
 }
+
+// PublicUser is only for userdata which every other user can see.
+type PublicUser struct {
```
Comment (dominikbraun) [inline services/user/types.go]: Shouldn't we name this `Profile` or something? I guess `user.Profile` sounds better than `user.PublicUser`

Comment (aligator) [reply services/user/types.go]: yes, but a Profile could also be from viewpoint of the user itself. It does not imply that it's public... maybe user.PublicProfile
Later we may have a PrivateProfile...

File: controllers/rest/user.go (hunk 1 above)
Comment (aligator) [reply controllers/rest/user.go]: you are right

**Pull request merged**